src/server/models/db/migrations/20230814130516_update_name_of_gambia/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- Update name of Gambia to The Gambia
+
+UPDATE "Country"
+SET name = 'The Gambia'
+WHERE name = 'Gambia';
```

Comment:
We also end up storing `country` in `jsonData.country` (we shouldn't..! 🤦) - so we should migrate those as well.

`where "jsonData"->>'country' = 'Gambia';`